src/Runtime/workflow-engine/docs/technical-guide.md

```
@@ -306,6 +308,34 @@ When `cascade=true`, all transitively dependent workflows in `DependencyFailed`
 
 Returns 404 if the workflow does not exist, or 409 if it is not in a resumable state (e.g. `Completed` or `Processing`).
 
+## Abandon
+
+An unsuccessful terminal workflow (`Failed`, `Canceled`, `DependencyFailed`) can be **abandoned** — its failure is explicitly written off by a caller:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)